submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestReleaser.kt

```diff
@@ -0,0 +1,21 @@
+package ac.uk.ebi.biostd.submission.submitter.request
+
+import ac.uk.ebi.biostd.persistence.common.service.SubmissionPersistenceService
+import ac.uk.ebi.biostd.persistence.filesystem.api.FtpService
+import ebi.ac.uk.extended.model.ExtSubmission
+import mu.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
+
+class SubmissionRequestReleaser(
```

Comment:
I think this name is misleading since it sounds like you're publishing the request and not the submission. Why not call it just`SubmissionReleaser`?